Base/usr/share/man/man5/ini.md

```diff
@@ -0,0 +1,21 @@
+## Name
+
+INI - generic config file format (.ini)
+
+## Description
+
+INI files serve as human-readable configuration files.
+
+INI files consist of key-value configuration data pairs separated by a unique group in square brackets.
```

Comment:
- INI files are not inherently configuration
- The group name is optional and defaults to the empty string in our parser

```suggestion
They consist of key-value pairs separated by '=', optionally located under a unique group in square brackets.
```